dhcpv4/ztpv4/parse_circuitid.go

```diff
@@ -0,0 +1,97 @@
+package ztpv4
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/insomniacslk/dhcp/dhcpv4"
+)
+
+// CircuitID represents the structure of network vendor interface formats
+type CircuitID struct {
+	Slot		string
+	Module	string
+	Port		string
+	SubPort string
+	Vlan		string
+}
+
+var circuitRegexs = []*regexp.Regexp{
+	// Juniper QFX et-0/0/0:0.0
+	regexp.MustCompile(".*(?P<slot>[0-9]+)/(?P<mod>[0-9]+)/(?P<port>[0-9]+):(?P<subport>[0-9]+).*$"),
```

Comment:
do we expect to search the circuit ID in substrings, or do exact matching? In the latter case the leading and trailing `.*` should be removed

Reply:
So, I have got these circuitid formats(regexs) from the KEA code. I can make the regex more exact when I test this with different network device. Will this do ?